frontend/tests/utils/helpers.ts

```diff
@@ -29,27 +63,31 @@ export async function goToPage(
 export async function login(page: Page): Promise<void> {
   await goToPage(undefined, page);
 
-  expect(process.env.TEST_ACCOUNT_PASS).toBeDefined();
-
-  const cookies = await (await page.context()).cookies();
+  const { username, password } = await getTestUsernameAndPassword();
 
-  if (cookies.length) return;
-
-  const username = TEST_USERNAME;
+  expect(username).toBeDefined();
 
   await page.click(getText("Log In"));
 
   await page.fill('[name="Username"], [name="email"]', username);
-  await page.fill('[name="Password"], [name="password"]', TEST_PASSWORD);
+  await page.fill('[name="Password"], [name="password"]', password);
 
   await page.click('[value="login"], [name="submit"]');
 
+  /**
+   * (thuang): This is needed to ensure the browser has been navigated back to
```

Comment:
Thank you for adding this comment! I didn't realize navigation relied on this piece of code, sorry about that! 😅